src/ts/richtext/utils/richTextHelpers.ts

```diff
@@ -0,0 +1,44 @@
+export const paragraphToWhitespace = (string: string): string => {
+  // Only the first paragraph tag
+  string = string.replace("<p>","");
+  // Replace subsequent paragraph tags with hard breaks
+  string = string.replace(/<p>/g,"<br>");
+  // Remove all paragraph closing tags
+  string = string.replace(/<\/p>/g,"");
+  return string;
+};
+
+export const getWords = (text: string): string[] => {
+  if (!text) {
+    return [];
+  }
+
+  return text
+    .trim()
+    .replace(/<(?:.|\n)*?>/gm, '')
+    .replace(/[^(\w| )]/g, '')
+    .split(/\s+/)
+    .filter(_ => _.length !== 0);
+};
+
+export const isTooLong = (value: string, maxWordLength: number): boolean => {
+  const wordLength = getWords(value).reduce((length, word) => {
```

Comment:
I appreciate that this is copy and pasted from elsewhere, but as far as I can see this calculates the total number of characters in all of the words in `value` rather than just a count of the number of words in `value`

Reply:
Yes, looks like it - it's a bit of validation from media-atom-maker most recently updated in https://github.com/guardian/media-atom-maker/pull/1127 . Naming is a bit rubbish - better would be something like `tooManyCharactersInAllWords`. 

It looks like there's an assumption based into the original logic that spaces aren't including in Youtube's video title character limit, which would be odd.

Reply:
No explanation of the odd logic in the original PR. I'd be inclined to go for a simple `.length` on the string `value`.

Reply:
I'm removing these functions for now as they're currently unused -when length validation is added I'll make it simpler.